book/website/collaboration/team-manual.md

```diff
@@ -0,0 +1,203 @@
+How is the lab involved in the department? How does the lab work with participants?
+- Publications. What needs to happen before a paper is published? what are the preprint policies? 
+What are the journal preferences? How does the lab deal with authorship({ref}`cm-aa`)? 
+see Liboiron et al. 2017  and [Dalmeet Singh Chawla 2018](https://www.science.org/content/article/assigning-authorship-research-papers-can-be-tricky-these-approaches-can-help) for author order discussions. 
+    - Consider academic citation practices. [Who do you choose to link and re-circulate in your work? Who gets erased? Who should you stop citing?](http://www.criticalethnicstudiesjournal.org/citation-practices)
+- Conferences: 
+How do you give a talk? 
+Which conference should be attended and why? 
+How are visualisations made accessible? 
+How is work in progress presented? 
+What is the conference budget (this should be seperate from the training budget in 18)? 
+Guidelines for PhD candidates could be at least 1-2 local/regional conferences during the first year(s) of PhD, and 1 major international one later in the project.
+- Finances: What grants pay for the lab? 
+What will/won't the lab pay for? 
+What grants can people apply for? 
+What to expect in terms of reimbursements? 
+- Grants: How to deal with awarded grants? 
+Should they be public to the whole lab or should only summaries and outcomes be available? 
+Who can apply to grants?
+- Mentorship and development: 
+How do you get feedback from others? 
+How do you collaborate in/outside of the lab? 
+What are the opportunities for development and mentoring. 
+How do you discuss career development? 
+See [Mentoring and Advising Contract Points of Agreement](https://docs.google.com/document/d/1oOPjjF8RJ1EOFvZMFAKe19hUVyQ5FRpS-tPV_uE1dcI/edit) or a [career template plan](https://www.skillsweaving.com/Downloads) as an example. How can you get mentoring outside of the lab?
+- Training: How are training opportunities formalised? 
+What is the miminum amount of personal development days per year? 
+What is the budget available? 
+- Where to go for help? 
+This can be support in/outside the lab, schedules of trainings, and contact persons. 
+- Reading list that lab members should read.
+- How do all lab members hold each other accountable? What are procedures to follow? 
+How to ensure that this accountability is not on individuals but the collective? (see also below)
+
+### Institutional / national guidelines that affect Team Manuals
+Institional and national policies and guidelines prevail over the Lab Manual. 
+- Ethics and Safety: What are the safety procedures? What are the ethic procedures? 
+Is there a research integrity office? 
+- Research Integrity and culture: Is there an institutional policy on this? 
+If there is a dispute, what would be the procedure? Can you contact Human Resources about this?
+- What are the guidelines for research with animals, humans, personal data etc? (see chapter ...)
```

Comment:
adding a comment here to highlight this as a TODO. Maybe the entire Guide for Ethical research?

```suggestion
- What are the guidelines for research with animals, humans, personal data etc? (see chapter ...)
```

Reply:
Also, etc is flagged by the latin checker - what do you usually turn this into?